repo/mk/js.deps.private.mk

```diff
@@ -49,6 +49,7 @@ node_modules_private: ## Refresh node_modules_private folder.
 		DEP_VSN=`$(CAT) "package.json" | $(JSON) "privateDependencies $${DEP_NAME}"`; \
 		DEPS="$${DEPS} $${DEP_NAME}@$${DEP_VSN}"; \
 	done; \
-	$(RM) node_modules_private/etc || true
-	$(RM) node_modules_private/lib || true
+	$(RM) node_modules_private/etc || true; \
+	$(RM) node_modules_private/lib || true; \
 	$(NPM) install --prefix node_modules_private --global $${DEPS}
+
```

Comment:
this is also unnecessary, but I'll take it as is
see https://github.com/tobiipro/support-firecloud/commit/20b94df6527165c7799e4b67677ea62fca547004